### ISSUE TITLE
Implement new filters with sql

### ISSUE THREAD

**Issue by JadedRain** [JadedRain/Incrementum#384]
Show how our updated filter system will work using sql.

* Create a demo page that allows the filtering of individual stocks
* Should be able to filter by  APPL, INTC, and NVDA using a search bar
* Demo page should display what stocks are being filtered by
* Display can use an example batch from the database but should use the database of stocks
